File: Data Structures & Algorithms/top-k-elements-in-list/submission-3.java
class Solution {
    public int[] topKFrequent(int[] nums, int k) {
	    int[] frequency = new int[2000];
        for(int num: nums)
            frequency[num+1000]++;
        List<Integer> helperArray = new ArrayList<>();
        for (int num : nums)
            helperArray.add(num);   
        helperArray.sort((a,b) -> frequency[b+1000] - frequency[a+1000]);
        List<Integer> ans = new ArrayList<>();
        int count = 0;
        for(int i = 0; i < helperArray.size() && count < k; i++){
            if(!ans.contains(helperArray.get(i))){
                ans.add(helperArray.get(i));
                count++;
            }
                
        }
        return ans.stream().mapToInt(i -> i).toArray();
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-3.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        ans = 0;
        nums.sort()
        for i in range(len(nums)):
            start = nums[i] 
            count = 1
            for j in range(i+1, len(nums), 1):
                if nums[j] == start + 1:
                    count += 1
                    start += 1        
            ans = max(ans, count)
    
        return ans
